### PR TITLE
added UDP for Joypad

### LINKED ISSUE #47: Use UDP Joystick Signals
There's a [test-app](https://github.com/sinix-dev/sinix/tree/feat/test-app) branch that contains an extra page named "Logs" which will log all the responses received on UDP server running on `41431` port. Joystick widgets of sinix-mobile should be sending data to this UDP server instead of WebSocket connection.

[udp-test-client.py](https://github.com/sinix-dev/sinix/blob/feat/test-app/src-tauri/udp-test-client.py) can be used to check whether the server is live or not.

sinix-mobile should be able to send joystick signal to sinix desktop through this UDP channel.

image

### PR DESCRIPTION
**Pull request by ritvij14**
@sanket143 pls have a look at the code, if it works for Joypad then I will make the same changes for other elements and push it on the same PR.

Fixes #47

### CODE REVIEW

File: lib/pages/discover.dart [hunk 1]
```diff
@@ -21,7 +21,7 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
   void scan() async {
     final String ip = await Wifi.ip;
     final String subnet = ip.substring(0, ip.lastIndexOf('.'));
-    final int port = 41431;
+    final int port = 41430;
```
Comment (sanket143) [inline lib/pages/discover.dart:24]: also update url in `store.dart`
```
- var url = "http://$ipAddr:41431/register"
+ var url = "http://$ipAddr:41430/channel/register"
```
```
- "ws://$ipAddr:41431/ws/${user.username}",
+ "ws://$ipAddr:41430/channel/ws/${user.username}",
```

**Review by sanket143 — COMMENTED** [review]
there were some route changes when we merged several server into one port. after the suggested changes, you should be able to test it with [feat/test-app](https://github.com/sinix-dev/sinix/tree/feat/test-app).

**Comment by ritvij14** [comment]
> there were some route changes when we merged several servers into one port. after the suggested changes, you should be able to test it with [feat/test-app](https://github.com/sinix-dev/sinix/tree/feat/test-app).

@sanket143 The issue on my side is that I am unable to setup Tauri itself, I am getting a lot of errors during its installation.

File: lib/pages/discover.dart (hunk 1 above)
Comment (ritvij14) [reply lib/pages/discover.dart:24]: @sanket143 updated `store.dart`

**Comment by sanket143** [comment]
oh, I've uploaded two more binaries. you can try testing with the test version [sinix_test_0.1.0-alpha.0_amd64.deb](https://github.com/sinix-dev/sinix/releases/tag/sinix-v0.1.0-alpha.0). lemme know if you still face any issues with the desktop.

**Pull request merged**